email-log.php

```diff
@@ -241,12 +351,9 @@ function display_content_callback() {
 	}
 
 	/**
-	 * Save Screen option.
+	 * Saves Screen options.
 	 *
-	 * @param unknown $status
-	 * @param unknown $option
-	 * @param unknown $value
-	 * @return unknown
+	 * @since Genesis
 	 */
 	function save_screen_options( $status, $option, $value ) {
```

Comment:
Looks like the documentation for this filter is wrong in WordPress documentation. You can find the correct documentation for this filter in the WordPress source files at https://core.trac.wordpress.org/browser/tags/4.5/src/wp-admin/includes/misc.php#L456